rdb/project.clj

```diff
@@ -0,0 +1,17 @@
+(defproject mongodb "0.1.0-SNAPSHOT"
+  :description "RethinkDB Jepsen Tests"
+;  :url "http://github.com/rethinkdb/jepsen"
```

Comment:
What should go here do you think?  All the other toplevel directories had `example.com/FIXME` here.